package.json

```
@@ -0,0 +1,23 @@
+{
+  "name": "messenger",
+  "version": "1.0.0",
+  "description": "",
+  "scripts": {
+    "dev": "parcel ./index.hbs --no-cache",
+    "build": "parcel build ./index.hbs",
+    "start": "parcel build ./index.hbs && node ./server.js"
```

Можно переиспользовать команды
```suggestion
    "start": "npm run build && node ./server.js"
```